sqllineage/config.py

```diff
@@ -3,32 +3,36 @@
 
 class _SQLLineageConfigLoader:
     """
-    Load all configurable items from environment variable, otherwise fallback to default
+    Load all configurable items from config variable, otherwise fallback to default
     """
 
-    # inspired by https://github.com/joke2k/django-environ
-    config = {
-        # for frontend directory drawer
-        "DIRECTORY": (str, os.path.join(os.path.dirname(__file__), "data")),
-        # set default schema/database
-        "DEFAULT_SCHEMA": (str, ""),
-        # to enable tsql no semicolon splitter mode
-        "TSQL_NO_SEMICOLON": (bool, False),
-    }
-    BOOLEAN_TRUE_STRINGS = ("true", "on", "ok", "y", "yes", "1")
+    def __init__(self) -> None:
+        self._init = False
 
-    def __getattr__(self, item):
-        if item in self.config:
-            type_, default = self.config[item]
-            # require SQLLINEAGE_ prefix from environment variable
-            return self.parse_value(
-                os.environ.get("SQLLINEAGE_" + item, default), type_
-            )
-        else:
-            return super().__getattribute__(item)
+        self._config = {
+            "DIRECTORY": {
+                "class_type": str,
+                "additional": None,
+                "default": os.path.join(os.path.dirname(__file__), "data"),
+            },
+            "DEFAULT_SCHEMA": {
+                "class_type": str,
+                "additional": None,
+                "default": None,
+            },
+            "TSQL_NO_SEMICOLON": {
+                "class_type": bool,
+                "additional": None,
+                "default": False,
+            },
+        }
```

Comment:
Let's not change the config data structure. Just create a separate variable to store the value set at runtime.